packages/app/src/app/components/DeploymentIntegration/DetailInfo/DetailInfo.tsx

```diff
@@ -10,7 +10,7 @@ export const DetailInfo = ({ info, deploy, bgColor, light, loading }) => (
       <Info light={light}>{info}</Info>
     </Margin>
 
-    <Button small disabled={loading} onClick={deploy}>
+    <Button small disabled={loading} onClick={() => deploy()}>
```

Comment:
This is quite important, otherwise you get this:

![image](https://user-images.githubusercontent.com/587016/61284486-55b49180-a7bf-11e9-8781-54ece60112ad.png)

or other subtle bugs. This is why I prefer an arrow func whenever we don't want to explicitly pass the event to the function.

Reply:
Will keep that in mind next time 🙂